src/opendr/perception/heart_anomaly_detection/attention_neural_bag_of_feature/algorithm/models.py

```diff
@@ -0,0 +1,349 @@
+# Copyright 2020-2021 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
```suggestion
```

Reply:
I don't know if it's the case, but unless the algorithm was modified by you, I would leave only the original copyright notice mentioned right after (provided by its developer).